emotesMain/src/main/resources/assets/emotecraft/lang/en_US.json

```diff
@@ -1,68 +1,68 @@
 {
-  "emotecraft.options.options": "Mod options",
-  "emotecraft.options.keybind": "Set a custom key to play the selected emote",
-  "emotecraft.options.fastmenu": "Select an emote than click to",
-  "emotecraft.options.fastmenu2": "to add to the fast-choose wheel.",
-  "emotecraft.options.fastmenu3": "Right click to clear",
+  "emotecraft.options.options": "Mod Options",
+  "emotecraft.options.keybind": "Set a key to play the selected emote",
+  "emotecraft.options.fastmenu": "Select an emote, then click on",
+  "emotecraft.options.fastmenu2": "the emote wheel to add it.",
+  "emotecraft.options.fastmenu3": "Right click an emote on the wheel to clear it.",
   "emotecraft.sure": "Overwriting keybind",
-  "emotecraft.sure2": "An other emote has this keybind.\nDo you want to reset that and set the key to this?",
-  "emotecraft.otherconfig.category.general": "Basic options",
-  "emotecraft.otherconfig.category.expert": "Expert options",
-  "emotecraft.otherconfig": "Other options",
+  "emotecraft.sure2": "Another emote is already bound to this key.\nDo you want overwrite that emote with this?",
+  "emotecraft.otherconfig.category.general": "Basic Options",
+  "emotecraft.otherconfig.category.expert": "Expert Options",
+  "emotecraft.otherconfig": "Other Options",
   "emotecraft.otherconfig.debug": "Debug",
-  "emotecraft.otherconfig.validate": "Validate emote",
-  "emotecraft.otherconfig.validate.tooltip": "Only for PvP worlds, where somebody want to cheat. It won't allow emotes, what makes too much offset",
-  "emotecraft.otherconfig.validationThreshold": "Validation threshold",
-  "emotecraft.otherconfig.validationThreshold.tooltip": "Exactly how much offset is too much",
-  "emotecraft.otherconfig.dark": "Dark fast-choose wheel",
-  "emotecraft.otherconfig.showicon": "Show icons instead of emote names",
-  "emotecraft.otherconfig.showicon.tooltip": "If available",
-  "emotecraft.otherconfig.stopthreshold": "Stop emote at this speed threshold",
-  "emotecraft.otherconfig.stopthreshold.tooltip": " (0.22 is walking speed)\nwith large value, you won't stop playing the emote. You can still use the stop play key",
-  "emotecraft.otherconfig.yratio": "Y axis moving influence in stop threshold",
-  "emotecraft.otherconfig.yratio.tooltip": "bigger value means bigger influence",
-  "emotecraft.otherconfig.loadbuiltin": "Load built-in emotes",
+  "emotecraft.otherconfig.validate": "Validate Emotes",
+  "emotecraft.otherconfig.validate.tooltip": "Mainly intended for PvP worlds. Disallows emotes that offset the player too much.",
+  "emotecraft.otherconfig.validationThreshold": "Validation Threshold",
+  "emotecraft.otherconfig.validationThreshold.tooltip": "How much offset is too much.",
+  "emotecraft.otherconfig.dark": "Use Dark Emote Wheel",
+  "emotecraft.otherconfig.showicon": "Show Icons Only (if available)",
+  "emotecraft.otherconfig.showicon.tooltip": "REMOVE THIS STRING? IDK",
```

Comment:
Just remove it!

https://github.com/KosmX/emotes/blob/c344d18d7f9c8f04a7b76016c0644540a94b893a/emotesMain/src/main/java/io/github/kosmx/emotes/main/config/ClientConfig.java#L16

And change the second bool to `false` to disable tooltip for this entry.

Reply:
Done! Sorry, I'm not completely familiar with this code base, and Minecraft modding in general.

Reply:
No problem.
Just what is if I don't read the changes and I just merge it with that string.

Reply:
No need, I fixed it myself [794a03f](https://github.com/KosmX/emotes/pull/254/commits/794a03fa6d164d1598884538eeac35fd4b3496b5) :)